src/main/resources/JavadocParser.g4

```diff
@@ -149,6 +150,9 @@ seeTag
     | SEE htmlElement description?
     ;
 
+
+serialFieldTag: SERIAL_FIELD fieldName=IDENTIFIER FIELD_TYPE  description?;
```

Comment:
This is another instance of reference.
https://github.com/openjdk/jdk/blob/8ea544c33fc502208577249fb83544f8d876bc17/src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java#L1442-L1448